### ISSUE TITLE
Post idea...List who has access to an SMB Share

### ISSUE THREAD

**Issue by mikeegan400** [PowerShell/Community-Blog#109]
Details of requested document:

- Are you offering to write the post [Y/N]: Y
- Proposed title: List who has access to an SMB Share
- Description of the problem/scenario/question you want covered: I have a blog entry ready that describes how to pull the ACL list for an SMB Share.  How do I submit the post for review?

Thank you,
Mike

**Comment by sdwheeler** [comment]
@mikeegan400 See our contributor guide https://github.com/PowerShell/Community-Blog/tree/main/Docs

**Comment by mikeegan400** [comment]
I am not seeing an 11 folder under the 2022 folder.  Do I need to create that folder?

Mike Egan
Senior Systems Engineer

***@***.***<http://www.kldiscovery.com/>
Service excellence powered by our transformative technology - Created for, and influenced by, our clients.
8995 Columbine Rd.
Eden Prairie, MN 55347
+1 952.358-5241 Office
+1 612-718-7240 Mobile
***@***.******@***.***>

www.kldiscovery.com<http://www.kldiscovery.com/>
KLDiscovery Ontrack, LLC., registered in Delaware with trading address at 9023 Columbine Road, Eden Prairie, MN 55347, USA.    For information on how we collect, process and retain your personal data, please see our Privacy Policy<https://www.kldiscovery.com/privacy-policy/>.

**Comment by sdwheeler** [comment]
@mikeegan400 Yes, create the folder in your PR.

**Comment by mikeegan400** [comment]
Sean,
I don't have much experience with GitHub.  I am assuming I need to follow the Submitting a Pull Request (PR) documentation in order to upload my blog post for review?

Mike

Mike Egan
Senior Systems Engineer

***@***.***<http://www.kldiscovery.com/>
Service excellence powered by our transformative technology - Created for, and influenced by, our clients.
8995 Columbine Rd.
Eden Prairie, MN 55347
+1 952.358-5241 Office
+1 612-718-7240 Mobile
***@***.******@***.***>

www.kldiscovery.com<http://www.kldiscovery.com/>
KLDiscovery Ontrack, LLC., registered in Delaware with trading address at 9023 Columbine Road, Eden Prairie, MN 55347, USA.    For information on how we collect, process and retain your personal data, please see our Privacy Policy<https://www.kldiscovery.com/privacy-policy/>.

**Comment by sdwheeler** [comment]
@mikeegan400 Yes, follow https://github.com/PowerShell/Community-Blog/blob/main/Docs/Submitting-a-PR.md. If you need to understand how the GitHub workflow works, see https://github.com/PowerShell/Community-Blog/blob/main/Docs/GitHub-workflow-for-new-post.md
